app/src/interfaces/list-o2m/list-o2m.vue

```diff
@@ -379,13 +379,10 @@ export default defineComponent({
 				const hasPrimaryKey = pkField in item;
 
 				const edits = (props.value || []).find(
-					(edit: any) =>
-						typeof edit === 'object' &&
-						relatedPrimaryKeyField.value?.field &&
-						edit[relatedPrimaryKeyField.value?.field] === item[relatedPrimaryKeyField.value?.field]
+					(edit: any) => typeof edit === 'object' && pkField && edit[pkField] === item[pkField]
```

Comment:
On second look, this `pkField` here doesn't check anything in the edit itself, as pkField is always a string of the name of the primary key field

Reply:
I agree with your review, however that code was already there. I only simplified it and made it more consistent with other uses of the same reference to `relatedPrimaryKeyField.value?.field`, such as in the `stageEdits` method. Perhaps I should have just left it alone since it doesn't related to the problem anyway.

With that in mind, do you want me to reverse it to the original state, fix it as you suggested, or keep it like this?

Reply:
If it's totally unrelated to the current fix, lets revert it to what it was before 👍🏻